spec/factories/course_forum_topics.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   factory :forum_topic, class: Course::Forum::Topic.name do
-    forum
+    transient do
+      course { build(:course) }
```

Comment:
Thanks ! adding a parent saves some code of defining the course, and if so we should also do the same thing for `course_assessment_answer` factory.